plugins/smartcard/msd-smartcard-plugin.c

```diff
@@ -217,14 +234,14 @@ process_smartcard_removal (MsdSmartcardPlugin *plugin)
 
         switch (remove_action)
         {
-            case MSD_SMARTCARD_REMOVE_ACTION_NONE:
-                return;
-            case MSD_SMARTCARD_REMOVE_ACTION_LOCK_SCREEN:
-                lock_screen (plugin);
-                break;
-            case MSD_SMARTCARD_REMOVE_ACTION_FORCE_LOGOUT:
-                force_logout (plugin);
-                break;
+                case MSD_SMARTCARD_REMOVE_ACTION_NONE:
+                        return;
+                case MSD_SMARTCARD_REMOVE_ACTION_LOCK_SCREEN:
+                        lock_screen (plugin);
+                        break;
+                case MSD_SMARTCARD_REMOVE_ACTION_FORCE_LOGOUT:
+                        force_logout (plugin);
+                        break;
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.